src/server/api/routes/api-router.js

```diff
@@ -34,9 +33,8 @@ const swaggerDocument = swaggerJsDoc(swaggerOptions);
 router.use('/documentation', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
 // Application routes
-router.use('/modules', modulesRouter);
 router.use('/channels', channelsRouter);
+router.use('/modules', modulesRouter);
 router.use('/messages', messagesRouter);
-router.use('/channel-members', channelMembersController);
```

Comment:
Why are you deleting this and the channel-members controller?? That is quite a lot of work.

Reply:
You are still deleting quite a lot here. You should fx not delete the channel embers router.

Reply:
I have a doubt here,  are the files like channel-members.controller.js, messages-controller.js, users-controller.js and their related files in the routes  be merged to my branch or be set to develop after I solve the merge conflicts?

Reply:
They have already been merged, so the channels controller where you do work does not need to be changed. And definitely not the channelMembersRouter